controllers/dellcsivolumegroupsnapshot_controller.go

```diff
@@ -473,6 +544,7 @@ func (r *DellCsiVolumeGroupSnapshotReconciler) createVolumeSnapshot(
 			failedMap[s.Name] = fmt.Sprintf("VG Snapshotter vg created ok unable to update VolumeSnapshotContent  %s in %s", contentName, vgName)
```

Comment:
same as earlier comment. This string occurs in several places

Reply:
we did not use any words like but/however , to make grep easy